src/components/ArticleInfo.jsx

```diff
@@ -0,0 +1,16 @@
+import "../App.css";
+
+export const ArticleInfo = ({ article }) => {
+  return (
+    <>
+      <section className="articleInfo">
+        <h2 className="title">{article.title}</h2>
+        <h3>Topic: {article.topic}</h3>
+        <p className="author"> Author: {article.author}</p>
+        <p className="dateMade">Written: {article.created_at}</p>
```

Comment:
The `created_at` value currently is a bit hard to read in the format it's in. To improve on this you could reformat the date using a util function, or try using a library like [Day.js](https://day.js.org/) to do the hard work for you 📆